src/main/java/org/apache/skywalking/banyandb/v1/client/StreamQuery.java

```diff
@@ -65,6 +65,8 @@ BanyandbStream.QueryRequest build() throws BanyanDBException {
                 .setMetadata(buildMetadata());
         if (timestampRange != null) {
             builder.setTimeRange(timestampRange.build());
+        } else {
+            builder.setTimeRange(TimestampRange.MAX_RANGE);
```

Comment:
Similar to https://github.com/apache/skywalking-banyandb-java-client/pull/13#discussion_r873292570

Reply:
Default time range for `Stream` has been removed, while it is still kept for `Measure`.